public/app/screens/main/devices/device-list/device-list.jsx

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var React = require("react");
+var Device = require('./device');
+var Room = require('./room');
+
+var DeviceList = React.createClass({
```

Comment:
Why does a device list encapsulate a room? Why do we need this layer of abstraction?